src/components/board/Stack.vue

```diff
@@ -42,11 +42,14 @@
 						value="">
 				</form>
 			</transition>
-			<Actions v-if="canManage && !isArchived" :force-menu="true">
-				<ActionButton icon="icon-archive" @click="modalArchivAllCardsShow=true">
+			<Actions v-if="!isArchived" :force-menu="true">
+				<ActionButton v-if="canManage" icon="icon-archive" @click="modalArchivAllCardsShow=true">
 					{{ t('deck', 'Archive all cards') }}
 				</ActionButton>
-				<ActionButton icon="icon-delete" @click="deleteStack(stack)">
+				<ActionButton v-if="canEdit" icon="icon-clone" @click="cloneStack(stack)">
```

Comment:
```suggestion
				<ActionButton v-if="canManage" icon="icon-clone" @click="cloneStack(stack)">
```

Creating new stacks requires manage permissions so we should do the same with cloning (since this is also what is checked in the backend)